api/application/client.go

```diff
@@ -580,6 +580,41 @@ func (c *Client) DestroyConsumedApplication(saasNames ...string) ([]params.Error
 	return allResults, nil
 }
 
+// ScaleApplicationParams contains parameters for the ScaleApplication API method.
+type ScaleApplicationParams struct {
+	// ApplicationName is the application to scale.
+	ApplicationName string
+
+	// Scale is the number of units which should be running.
+	Scale int
+}
+
+// DestroyUnits decreases the number of units dedicated to one or more
+// applications.
+func (c *Client) ScaleApplication(in ScaleApplicationParams) (params.ScaleApplicationResult, error) {
```

Comment:
Personally, I am not too excited to see apiserver.params to be returned from api methods. Admittedly, there is precedence and we are not in my ideal world :)

Reply:
Yeah, I agree. In this case was sticking with convention more than anything.